app/adapters/job.js

```diff
@@ -13,6 +13,17 @@ export default ApplicationAdapter.extend({
   restart(id) {
     return this.ajax(this.urlForJobControlAction(id, 'restart'), 'POST');
   },
+  authorize(id, token) {
+    const payload = {
+      "token": token
+    };
+    //our adapter specifies rootobject format so we must put payload inside a job
+    return this.ajax(this.urlForJobControlAction(id, 'authorize'), 'POST', {
+      data: {
+        jobs: payload
```

Comment:
The JSON object POSTed here will look something like like `{"jobs":{"token":"abc"}}`, which is unexpected, because this object does not contain a `job`.

Assuming this decision was just to satisfy the [parser](https://github.com/Duke-GCB/bespin-api/blob/9953d5c1815f6193785270b93ef5d68b5f27e3a4/data/parsers.py#L33), I'd rather we override the serializer for the `authorize` detail route, like [`create-job`](https://github.com/Duke-GCB/bespin-api/blob/9953d5c1815f6193785270b93ef5d68b5f27e3a4/data/api.py#L235). 

We wouldn't need to pretend the payload is a `job`, but would need a serializer for the authorize token, which I think is reasonable.